manager/integration/tests/common.py

```diff
@@ -3129,10 +3129,10 @@ def wait_for_backup_volume_backing_image_synced(
     completed = False
     for _ in range(retry_count):
         bv = find_backup_volume(client, volume_name)
-        assert bv is not None
-        if bv.backingImageName == backing_image:
-            completed = True
-            break
+        if bv is not None and bv.created != "":
```

Comment:
Do we need  the condition `and bv.created != ""`? in your new statement? since the return of `find_backup_volume` already included it?

Reply:
The condition `and bv.created != ""` is redundant. I have already fixed that.